package-lock.json

```diff
@@ -9,6 +9,7 @@
       "version": "0.1.0",
       "dependencies": {
         "@next/font": "^14.2.4",
+        "codemod": "^0.11.22",
```

Comment:
you manually removed codemod from package.json. please don't do that :)
instead, just run `npm uninstall codemod`